src/Symfony/Component/Console/Descriptor/MarkdownDescriptor.php

```diff
@@ -142,4 +168,17 @@ protected function describeApplication(Application $application, array $options
             $this->write($this->describeCommand($command));
         }
     }
+
+    private function getApplicationTitle(Application $application)
```

Comment:
If we do a new method i'd say `ApplicationDescription::getTitle` for better reuseability.